.github/workflows/codeql.yml

```diff
@@ -13,11 +13,11 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "support_openssl_3.1*" ]
```

Comment:
Do we need to revert the changes from Line 16 to Line20?

Reply:
Yes, that makes sense.
I'll update the PR.